### DISCUSSION TITLE
How to Enable and active Mattermost Login Authentication

### DISCUSSION THREAD

**Discussion by ShaonPro** [outline/outline#7832] [Self Hosting]
I want to enable this for my team as my team using mattermost and my organization emails both are with same emails. 
Want to integrate mattermost using OIDC, i have tried but it's not working perfectly. 
<img width="652" alt="Screenshot 2024-10-25 at 7 12 14 PM" src="https://github.com/user-attachments/assets/5d3b6dbe-8245-4a36-aeed-d432ca4ecc98">

Is there any help available?

**Comment by tommoor** [comment]
Please post the logs from the server so we can see the issue. I'm not sure if Mattermost is fully compatible or not as an OIDC provider

**Reply by ShaonPro** [reply]
@tommoor 
ERR Error during authentication 

As per this i a using Mattermost
https://docs.getoutline.com/s/hosting/doc/oidc-8CPBm6uC0I

**Reply by ShaonPro** [reply]
@tommoor 

`$ node ./build/server/index.js
{"label":"lifecycle","level":"info","message":"Note: Restricting process count to 1 due to use of collaborative service"}
{"label":"lifecycle","level":"info","message":"\nIs your team enjoying Outline? Consider supporting future development by sponsoring the project:\n\nhttps://github.com/sponsors/outline\n"}
{"label":"lifecycle","level":"info","message":"Note: Restricting process count to 1 due to use of collaborative service"}
{"label":"lifecycle","level":"info","message":"Starting collaboration service"}
{"label":"lifecycle","level":"info","message":"Starting websockets service"}
{"label":"lifecycle","level":"info","message":"Starting worker service"}
{"label":"lifecycle","level":"info","message":"Starting web service"}
{"level":"warn","message":"Enforced https was disabled with FORCE_HTTPS env variable"}
{"label":"lifecycle","level":"info","message":"Listening on http://localhost:3000 / https://wiki.thedevgarden.icu"}
{"level":"error","message":"Error during authentication"}
{"level":"error","message":"Error during authentication"}`

**Reply by tommoor** [reply]
Nothing useful there, try with some higher log levels `LOG_LEVEL=debug` or `LOG_LEVEL=silly`

**Reply by ShaonPro** [reply]
@tommoor - I have Remove that and start using emails only.
